packages/client/wallets/smart-wallet/src/blockchain/wallets/EVMSmartWallet.ts

```diff
@@ -110,4 +123,71 @@ export class EVMSmartWallet {
     public async nfts() {
         return this.crossmintService.fetchNFTs(this.address, this.chain);
     }
+
+    /**
+     * Sends a contract call transaction and returns the hash of a confirmed transaction.
```

Comment:
Need to update naming in the jsdoc still!

Reply:
Good catch